lib/main_common.pm

```diff
@@ -481,7 +481,6 @@ sub load_zdup_tests {
     # Restrict version switch to sle until opensuse adopts it
     loadtest "migration/version_switch_upgrade_target" if is_sle and get_var("UPGRADE_TARGET_VERSION");
     loadtest 'boot/boot_to_desktop';
-    loadtest "installation/opensuse_welcome"  if opensuse_welcome_applicable();
```

Comment:
This cannot be done until **first_boot** handles the popup: https://openqa.opensuse.org/tests/1234107

Or all those tests will fail

Reply:
> This cannot be done until **first_boot** handles the popup: https://openqa.opensuse.org/tests/1234107#step/opensuse_welcome/5
> 
> Or all those tests will fail

actually this is working for opensuse_welcome, confusing...